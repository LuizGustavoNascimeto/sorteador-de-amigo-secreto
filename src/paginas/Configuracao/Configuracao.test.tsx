import { render, screen } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import { Configuracao } from ".";

const mockNavigacao = jest.fn();
jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockNavigacao,
  };
});

describe("a pagina de configuração", () => {
  test("deverá ser renderizada corretamente", () => {
    const { container } = render(
      <RecoilRoot>
        <Configuracao />
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
