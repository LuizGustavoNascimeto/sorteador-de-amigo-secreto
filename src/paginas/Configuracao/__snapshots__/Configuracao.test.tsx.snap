// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuração deverá ser renderizada corretamente 1`] = `
<div>
  <form>
    <input
      name="participante"
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Começar brincadeira
    </button>
  </footer>
</div>
`;
